src/OscapScannerBase.cpp

```diff
@@ -383,6 +383,9 @@ bool OscapScannerBase::tryToReadStdOutChar(QProcess& process)
                         emit warningMessage(QString("Failed to download \"%1\"!").arg(mLastDownloadingFile));
                 }
                 break;
+            default:
```

Comment:
This has symmetry with the above (hidden in diff) block, and based upon it not affecting current operations, I'm going to assume this is correct, but I'd appreciate someone more familiar with the code to validate that it is.